lib/std/Target.zig

```diff
@@ -2164,6 +2176,7 @@ pub const DynamicLinker = struct {
             .ps3,
             .ps4,
             .ps5,
+            .vita,
             => .none,
```

Comment:
There's no interpreter path embedded in executables at all?

Reply:
Vita doesn't have interpreter, everything is handled by kernel module.